src/subgraph/convert.c

```diff
@@ -530,7 +530,8 @@ enum xnn_status xnn_define_convert(
   if ((flags & XNN_FLAG_MAYBE_PACK_FOR_GEMM) &&
       input_value->datatype == xnn_datatype_fp32 &&
       output_value->datatype == xnn_datatype_qdint8 &&
-      xnn_init_qp8_f32_qc4w_gemm_config() != NULL) {
+      (xnn_init_qp8_f32_qc4w_gemm_config() != NULL || 
```

Comment:
Also, I think the logic is not correct since the `xnn_init_qp8_f32_qc4w_gemm_config()` and `xnn_init_qp8_f32_qb4w_gemm_config` would have to be matched to the `input_datatype`.